composer.json

```diff
@@ -3,24 +3,24 @@
     "license" : "GPL-3.0+",
     "description" : "LORIS (Longitudinal Online Research and Imaging System) is a web-accessible database solution for neuroimaging.",
     "require" : {
-        "smarty/smarty" : "~3.1.33",
+        "smarty/smarty" : "~3.1.36",
         "firebase/php-jwt" : "~3.0",
         "google/recaptcha": "~1.1",
-        "php-http/guzzle6-adapter": ">=2.0.0",
+        "php-http/guzzle7-adapter": "0.1.1",
         "psr/http-message": "~1.0",
         "psr/http-server-handler" : "*",
         "psr/http-server-middleware" : "*",
-        "laminas/laminas-diactoros" : "~1.8.4",
+        "laminas/laminas-diactoros" : "2.5.0",
         "ext-json": "*",
-        "bjeavons/zxcvbn-php": "^0.4"
+        "bjeavons/zxcvbn-php": "^1.0"
     },
     "require-dev" : {
-        "squizlabs/php_codesniffer" : "3.5.3",
-        "phpunit/phpunit" : "7.0.0",
-        "phan/phan": "3.0.0",
+        "squizlabs/php_codesniffer" : "3.5.8",
+        "phpunit/phpunit" : "9.4.4",
+        "phan/phan": "4.0.0-alpha5",
```

Comment:
3.2.6 will work with php8, you don't need to update to an alpha version:

https://github.com/phan/phan/blob/e14110d4bef8643562b02a4003015c2c0dcc9fe4/composer.json#L25